CONTRIBUTING.md

```diff
@@ -285,8 +285,11 @@ We avoid breaking changes unless absolutely necessary and unavoidable
  8. Change must be well documented (build / runtime test logs, pr, git
     commit, documentation, release notes, etc) with clear notes on how to
     fix the introduced problems.
- 9. Breaking Change must be clearly marked with a `[BREAKING]` tag in the
-    git commit topic and PR title that will propagate to Release Notes.
+ 9. Breaking Change must be clearly identified with inclusion of the an
```

Comment:
9. Breaking Changes must be clearly marked both in Git Commits and Pull Requests: 1. Put exclamation mark `!` as first character of the title; 2. Put `BREAKING CHANGE:` in the body with a brief change description and the quick-fix instructions. This helps users identify areas that need an update on their side, grep git logs, and auto-generate Release Notes.

@hartmannathan could you please verify the grammar? :-P